moto/iam/config.py

```diff
@@ -0,0 +1,173 @@
+import json
+
+from moto.core.exceptions import InvalidNextTokenException
+from moto.core.models import ConfigQueryModel
+from moto.iam import iam_backends
+
+
+class RoleConfigQuery(ConfigQueryModel):
+    def list_config_service_resources(
+        self,
+        resource_ids,
+        resource_name,
+        limit,
+        next_token,
+        backend_region=None,
+        resource_region=None,
+    ):
+        # For aggregation -- did we get both a resource ID and a resource name?
+        if resource_ids and resource_name:
+            # If the values are different, then return an empty list:
```

Comment:
This logic is true for S3, but is it true for IAM roles?

I know that Roles have an ID that is some `AROAxxx` format, and not the name of the role.

Reply:
it wasn't; removed